docker-healthcheck.sh

```diff
@@ -1,13 +1,21 @@
 #!/bin/sh
 
 # Using Unix socket connection to test readiness.
-queryResult="$(psql --username=blackduck --dbname=bds_hub --port=5432 --tuples-only --no-align --quiet -c 'SELECT 1')"
+pg_isready --username=blackduck --dbname=bds_hub --port=5432 --quiet
+queryResult=$?
+
+if [ "$queryResult" == "0" ];
+then
+  # PostgreSQL is accepting connections.
+  exit 0
+fi
+
 if [ "$queryResult" == "1" ];
 then
-  # Successful PostgreSQL Unix socket connection and query result.
+  # PostgreSQL is still starting up.
   exit 0
 fi
 
-# Unsuccessful PostgreSQL Unix socket connection or query result.
+# PostgreSQL is not responding.
 exit 1
```

Comment:
We may want to return or at least echo the details of the pg_isready result for any non-0 exit code as this information is useful for debugging and captured via the healthcheck inspection in Dodkcer.

Reply:
OK.